app/screens/ScheduleScreen/ScheduleCard.tsx

```diff
@@ -79,7 +76,10 @@ const baseSpeakingEventProps = (heading: string, subheading: string) => ({
 })
 
 const ScheduleCard: FC<ScheduleCardProps> = observer(function ScheduleCard(props) {
-  const { variant = "event", onPress, time, eventTitle, heading, subheading } = props
+  const { variant = "event", time, eventTitle, heading, subheading } = props
+  const navigation = useAppNavigation()
+  const onPress =
+    ["talk", "workshop"].includes(variant) && (() => navigation.navigate("EventDetail"))
```

Comment:
This route should be `TalkDetails` (it got updated after you merged)

Reply:
fixed in latest push: https://github.com/infinitered/ChainReactApp2023/pull/10/commits/b85c02448b18e5bfed1aa49bd398609e65f9bf26